tagstudio/src/qt/ts_qt.py

```diff
@@ -90,7 +92,7 @@
     FixDupeFilesModal,
     FoldersToTagsModal,
 )
-import src.qt.resources_rc
```

Comment:
Removing this import is what causes in the splash screen to break, and is currently the pinpoint of a merge conflict

Reply:
Ah so it's not unused import, but it has side-effect of executing code. Not a good thing to do, but looks like there's no way around it. I'll add comment so it's obvious why is it here.